frameworks/Kotlin/http4k/core/src/test/kotlin/Http4kBenchmarkServerTest.kt

```diff
@@ -0,0 +1,56 @@
+
+import com.natpryce.hamkrest.should.shouldMatch
+import io.mockk.every
+import io.mockk.mockk
+import org.http4k.core.Method.GET
+import org.http4k.core.Request
+import org.http4k.format.Jackson.number
+import org.http4k.format.Jackson.obj
+import org.http4k.hamkrest.hasBody
+import org.junit.jupiter.api.Test
+import java.sql.Connection
+import java.sql.PreparedStatement
+
+class Http4kBenchmarkServerTest {
```

Comment:
We don't want to maintain any framework-specific functional tests in this repository.  Could you remove this file from the PR?

I made a similar request in a different PR recently and gave a longer explanation:  https://github.com/TechEmpower/FrameworkBenchmarks/pull/3895#discussion_r198972038